### PR TITLE
OKR: Mid-Quarter scoring

### PR DESCRIPTION
**Pull request by mikeal**
Heya @vmx and @warpfork, time for mid-quarter scoring :)

### CODE REVIEW

**Comment by warpfork** [comment]
Really wish there was such a thing as a two-quarter goal.

It would've been nice if I could've broken down "go-ipld-prime is a usable replacement for go-ipld-cbor" at the beginning quarter -- but also impossible; I would've had no idea how, and been wrong.

But I probably could've said I'd expect it to be 0.5 done by eoq.  (And so when I report it's on track for about 0.7 done, it's actually *way* above expectations...)

EDIT: I'm dumb, that's what the "projected score" column is for, isn't it...

**Review by warpfork — APPROVED** [review]

**Comment by mikeal** [comment]
@warpfork that’s a good point. I’m going to think about a way to potential do longer term OKR’s when we start planning Q2’s work. we could potentially merge/align much more directly with the roadmap doc as well.

File: okrs/2019-q1.md [hunk 1]
```diff
@@ -6,29 +6,29 @@ Background:
...
 
 ## Objective: Interfaces are in-line with current IPLD specifications and vision.
 
 | Key Result | Assignee | Priority | Mid-Q Score | Projected Score | Final Score |
 | ---------- | -------- | -------- | ----------- | --------------- | ----------- |
-| `go-ipld-prime` is a usable replacement for `go-ipld-cbor`. | @warpfork | ![p1](https://ipfs.io/ipfs/QmV88khHDJEXi7wo6o972MZWY661R9PhrZW6dvpFP6jnMn/p1.svg) | | | |
-| `js-ipld-interace` refactoring is complete. | @vmx | ![p1](https://ipfs.io/ipfs/QmV88khHDJEXi7wo6o972MZWY661R9PhrZW6dvpFP6jnMn/p1.svg) | | | |
-| Multiformat API's are easier to use with IPLD. | @vmx | ![p2](https://ipfs.io/ipfs/QmV88khHDJEXi7wo6o972MZWY661R9PhrZW6dvpFP6jnMn/p2.svg) | | | |
-| `ipfs-block` API is moved to IPLD project.| @mikeal | ![p2](https://ipfs.io/ipfs/QmV88khHDJEXi7wo6o972MZWY661R9PhrZW6dvpFP6jnMn/p2.svg) | | | |
+| `go-ipld-prime` is a usable replacement for `go-ipld-cbor`. | @warpfork | ![p1](https://ipfs.io/ipfs/QmV88khHDJEXi7wo6o972MZWY661R9PhrZW6dvpFP6jnMn/p1.svg) | 0.6 | 0.5 | |
```
Comment (mikeal) [inline okrs/2019-q1.md:18]: hrm... are you planning to delete some of the work you completed @warpfork? :)

Comment (warpfork) [reply okrs/2019-q1.md:18]: Hopefully not!  I was ... reading that as projections I would've made... earlier.  Are they supposed to be midq projections for eoq?  xD

Comment (mikeal) [reply okrs/2019-q1.md:18]: Yup :) It’s a way to surface early if we think something will take longer than we originally anticipated.

**Pull request merged**